search.json

```diff
@@ -35,4 +35,14 @@ layout:
 "date":"{{ post.date }}"
 } {% unless forloop.last %},{% endunless %}
```

Comment:
This line is the one I'm referring to- this will always be a comma, so the `unless` condition applies to the next block.